labs/src/main/scala/org/scalalabs/basic/lab03/RecursionPatternMatchingExercise.scala

```diff
@@ -29,7 +29,12 @@ object RecursionPatternMatchingExercise {
    * checkValuesIncrease(Seq(1,2,2)) == false
    */
   def checkValuesIncrease(seq: Seq[Int]): Boolean = {
-    error("fix me")
+    seq match {
+      case Nil => true
+      case _::Nil => true
```

Comment:
Yeah, you can pattern match on `List(_)`. You can also match on `List(1,2,x,3)` or even `List(x, _*)` which is equivalent to `x::_`. Finally, you can be tricksy and do `List(x, xs@_*)` which is equivalent to `x::xs`, because for some reason `List(x, xs*)` doesn't work.

You can also pattern match other collection types this way, such as `Seq(a,b)` matching a two-element Seq.